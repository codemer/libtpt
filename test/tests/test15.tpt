@if (1) {
	This should display.
} @else {
	This should not display.
}
@if (0) {
	This should not display.
} @elsif (1) {
	This should display.
} @elsif (0) {
	This should not display.
} @elsif (0) {
	This should not display.
} @else {
	This should not display.
}
@if (0) {
	This should not display.
} @elsif (0) {
	This should not display.
} @else {
	This should display.
}
