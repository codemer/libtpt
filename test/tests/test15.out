	This should display.
	This should display.
	This should display.
